client/src/components/Atoms/StatusBox.tsx

```diff
@@ -423,7 +423,7 @@ export const JurisdictionAdminStatusBox: React.FC<IJurisdictionAdminProps> = ({
   const inProgressHeadline = `Round ${roundNum} of the audit is in progress.`
 
   // Round in progress, hasn't set up audit boards
-  if (auditBoards.length === 0)
+  if (auditBoards.length === 0 && auditType !== 'BATCH_COMPARISON')
```

Comment:
Arlo would automatically divvy up the sampled batches between the audit boards and list those assignments in the batch retrieval list (and on the batch tally sheets). This followed the pattern from ballot polling/comparison audits, where Arlo divvies up the ballots between audit boards. It's also good for tracking who audited which ballots to keep a good record of the audit. Though since the audit boards weren't actually logging in for batch audits, that record didn't necessarily correspond to what happened in the actual audit - we know that some jurisdictions didn't follow those assignments.

Reply:
Got it! Thanks for that context